src/handlers.js

```diff
@@ -234,19 +234,22 @@ class Handlers {
 
   // `pointermove` and `mousemove` event handler
   handlePointerMove(event) {
+    let events = [];
     if (this._captureCoalescedEvents && event.getCoalescedEvents) {
-      event.getCoalescedEvents().forEach((coalescedEvent) => {
-        this.handleSinglePointerMove(coalescedEvent);
-      });
-    } else {
-      this.handleSinglePointerMove(event);
+      events = event.getCoalescedEvents();
+    }
+    if (events.length === 0) {
+      events = [event];
     }
+    events.forEach((coalescedEvent, index) => {
+      this.handleSinglePointerMove(coalescedEvent, index !== events.length - 1);
+    });
   }
 
-  handleSinglePointerMove(event) {
+  handleSinglePointerMove(event, coalesced) {
```

Comment:
This could use a little bit of improvement if this function is merged into `handleMouseMove` -- moving the `case` down there.

Reply:
I think I cannot move it there because, handleMouseMove is the event handler for mouse events (when pointerEvents are not supported, and pointerType is not defined for mouseEvents.

Reply:
ok!